templates/nginx_internal.conf

```diff
@@ -0,0 +1,72 @@
+worker_processes 1;
```

Comment:
Is there any reason to not just use `auto` here?

Reply:
Gotcha, I'll do that.